lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/RemoteConfiguration.java

```diff
@@ -132,4 +139,12 @@ private RemoteConfiguration() {
   public static final class RemoteServerStage implements Name<EStage<TransportEvent>> {
     // Intentionally empty
   }
+
+  /**
+   * Option for use http.
+   */
+  @NamedParameter(doc = "Option for use http.", default_value = "" + PROTOCOL_NETTY)
```

Comment:
It was used across the original source. Is it better to update to `String.valueOf` every use?